docs/code-quality/ca1021.md

```diff
@@ -85,5 +87,15 @@ Methods that implement the **Try\<Something>** pattern, such as <xref:System.Int
 ### Code
 [!code-csharp[FxCop.Design.TryPattern#1](../code-quality/codesnippet/CSharp/ca1021-avoid-out-parameters_5.cs)]
 
+## Configurability
```

Comment:
Please move the `Configurability` section above all the `Example` sections for ordering consistency with existing documentation pages.